R/auth.R

```diff
@@ -51,26 +55,23 @@ loginUrlFor <- function(appUrl, appServer) {
   }
 }
 
-
-# Returns the cookie to be persisted in all future HTTP requests in the session
+# Returns the cookies to be persisted in all future HTTP requests in the session
 handlePost <- function(handle, loginUrl, postfields, cookies, cookieName) {
   curl::handle_setopt(handle,
     postfields = postfields,
-    # TODO how should the cookies be encoding?
     cookie = pasteParams(cookies, "; "),
     post = TRUE,
     followlocation = FALSE,
-    ssl_verifyhost = 0, ssl_verifypeer = 0
+    ssl_verifyhost = 0,
+    ssl_verifypeer = 0
```

Comment:
Do you think ignoring SSL errors should be the default, or opt-in? (I could probably go either way, but ideally we wouldn't change it to be stricter once it's public)